### PR TITLE
Fixed lock withdrawal prompt not correct

### PR DESCRIPTION
**Pull request by peterwillcn**
Signed-off-by: Xiaobo Tian <peterwillcn@gmail.com>

### CODE REVIEW

File: cr/state/committee.go [hunk 1]
```diff
@@ -526,6 +526,10 @@ func (c *Committee) shouldCleanHistory() bool {
 		c.params.CRDutyPeriod-c.params.CRVotingPeriod
 }
 
+func (c *Committee) IsDepositLockupBlocks(height uint32) bool {
+	return height < c.params.CRVotingStartHeight+c.params.CRDepositLockupBlocks
```
Comment (RainFallsSilent) [inline cr/state/committee.go]: lock time start from CRVotingStartHeight ?

File: blockchain/txvalidator.go [hunk 2]
```diff
@@ -2339,6 +2339,10 @@ func (b *BlockChain) checkReturnCRDepositCoinTransaction(txn *Transaction,
 		availableValue += b.crCommittee.GetAvailableDepositAmount(*cid)
 	}
 
+	if b.crCommittee.IsDepositLockupBlocks(currentHeight) {
```
Comment (RainFallsSilent) [inline blockchain/txvalidator.go]: no need to check here

File: blockchain/txvalidator.go [hunk 3]
```diff
@@ -2330,6 +2330,11 @@ func (b *BlockChain) checkReturnCRDepositCoinTransaction(txn *Transaction,
 		}
 
 		availableValue += b.crCommittee.GetAvailableDepositAmount(*cid)
+
+		c := b.crCommittee.GetCandidate(*cid)
```
Comment (Houshoupei84) [inline blockchain/txvalidator.go]: c maybe is nil

**Comment by peterwillcn** [comment]
This logic in getAvailableDepositAmount already exists, duplicate code. So let's close this pr.

**Pull request closed without merging**